Marlin/src/pins/lpc1769/pins_XTLW_CLIMBER_8TH_LPC.h

```diff
@@ -0,0 +1,280 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2020 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
```

Comment:
These should all be (c) 2024

Reply:
I don't agree with this, the original author added it to Marlin on the years provided

Reply:
Whenever pins/files are added here, they get the current year added to them.